utils/build-script

```diff
@@ -21,8 +21,10 @@ import textwrap
 
 sys.path.append(os.path.dirname(__file__))
 
-from SwiftBuildSupport import *
-
+from SwiftBuildSupport import check_call, get_all_preset_names
+from SwiftBuildSupport import get_preset_options, HOME, print_with_argv0
+from SwiftBuildSupport import quote_shell_command, SWIFT_BUILD_ROOT
+from SwiftBuildSupport import SWIFT_SOURCE_ROOT
```

Comment:
nit: How about instead:

```python
from SwiftBuildSupport import (
    SWIFT_SOURCE_ROOT,
    SWIFT_BUILD_ROOT,
    print_with_argv0,
    quote_shell_command,
    check_call,
    get_preset_options,
    get_all_preset_names
)
```

I've seen this `import` style suggested in a few style guides:

- https://www.python.org/dev/peps/pep-0328/
- http://docs.python-guide.org/en/latest/writing/style/